spec/rails_helper.rb

```diff
@@ -82,9 +82,9 @@
   config.include ControllerMacros, type: :feature
   config.include ControllerMacros, type: :request
   # config.include SelecterHelper,   type: :feature
-  # config.include SupportHelper,    type: :controller
-  # config.include SupportHelper,    type: :feature
-  # config.include SupportHelper,    type: :request
+  config.include SupportHelper,    type: :controller
+  config.include SupportHelper,    type: :feature
+  config.include SupportHelper,    type: :request
```

Comment:
```suggestion
  config.include SupportHelper, type: :controller
  config.include SupportHelper, type: :feature
  config.include SupportHelper, type: :request
```

Reply:
I uncommented and left the spacing for another day, at least we got Pro parity now.